include/core/Variant.hpp

```diff
@@ -74,6 +74,42 @@ class Variant {
 
 	};
 
+	enum Operator {
+
+		//comparation
+				OP_EQUAL,
```

Comment:
weird indentation 😛

Reply:
It was copied from godot sources. I will check and fix it :smile: